src/components/Navigation/index.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import Sidebar from 'react-sidebar';
+import { Icon } from '../../components/Icons';
+import CategoryList from '../../components/CategoryList';
+import { colors, sizes, media } from '../../helpers/styles';
+
+const NavigationFull = styled.div`
+    display: none;
+    @media (max-width: 767px) {
+        display: flex;
+    }
+
+    & .menu-mobile-box {
+        max-width: 80%;
+    }
+`;
+
+const MenuBtn = styled.button`
+    background: none;
+    border: none;
+    cursor: pointer;
+    position: relative;
+    margin-right: 15px;
+    line-height: 0;
+`;
+
+const IconMenu = styled(Icon)`
+    & path {
+        stroke: ${colors.primary};
+        transform: translate(-15px, -40px);
+        stroke-width: 3;
+    }
+`;
+
+class Navigation extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            sidebarOpen: false,
+        };
+        this.onSetSidebarOpen = this.onSetSidebarOpen.bind(this);
+    }
+
+    onSetSidebarOpen(open) {
+        this.setState({ sidebarOpen: open });
+    }
+
+    render() {
+        return (
+            <NavigationFull>
+                <Sidebar
+                    sidebar={<CategoryList />}
+                    open={this.state.sidebarOpen}
+                    onSetOpen={this.onSetSidebarOpen}
+                    rootClassName={'menu-mobile-box'}
```

Comment:
Já que vc está utilizando valor estático, não precisa das chaves, pode ser assim:
```suggestion
                    rootClassName="menu-mobile-box"
```